libs/providers/go-feature-flag/src/lib/go-feature-flag-provider.ts

```diff
@@ -117,11 +118,11 @@ export class GoFeatureFlagProvider implements Provider {
    */
   async callGoffDataCollection() {
     if (this.dataCollectorBuffer?.length === 0) {
-      return
+      return;
     }
 
-    const dataToSend = structuredClone(this.dataCollectorBuffer);
-    this.dataCollectorBuffer = []
+    const dataToSend = _.cloneDeep<FeatureEvent<any>[] | undefined>(this.dataCollectorBuffer) || [];
```

Comment:
Could you use `JSON.parse(JSON.stringify())`? That should work if `FeatureEvent` doesn't include circular references, maps, or sets.